### PR TITLE
More Party Name Changes

### PR DESCRIPTION
**Pull request by ccfox**
-Berlin
-Quebec
-Prussia
-A nationalist party for the GCC

### CODE REVIEW

File: CWE/common/countries/Quebec.txt [hunk 1]
```diff
@@ -45,51 +89,150 @@ party = {
...
+		economic_policy = state_capitalism
+		ideology = socialist
+		religious_policy = secularized
+		trade_policy = protectionism
+		war_policy = pacifism
+		start_date = 1955.1.1
+		end_date = 1961.1.1
+}
+party = {
+		name = "Parti Québécois"
```
Comment (settintotrieste) [inline CWE/common/countries/Quebec.txt:114]: No special characters please

File: CWE/common/countries/Quebec.txt [hunk 2]
```diff
@@ -45,51 +89,150 @@ party = {
...
+		economic_policy = interventionism
+		ideology = progressive
+		religious_policy = secularized
+		trade_policy = protectionism
+		war_policy = pacifism
+		start_date = 1955.1.1
+		end_date = 1961.1.1
+}
+party = {
+		name = "Parti Québécois (SL)"
```
Comment (settintotrieste) [inline CWE/common/countries/Quebec.txt:147]: No special characters please

File: CWE/common/countries/Quebec.txt [hunk 3]
```diff
@@ -45,51 +89,150 @@ party = {
...
+		economic_policy = interventionism
+		ideology = progressive
+		religious_policy = secularized
+		trade_policy = protectionism
+		war_policy = pacifism
+		start_date = 1961.1.1
+		end_date = 2006.2.4
+}
+party = {
+		name = "Québec solidaire (SL)"
```
Comment (settintotrieste) [inline CWE/common/countries/Quebec.txt:158]: No special characters please

File: CWE/common/countries/Quebec.txt [hunk 4]
```diff
@@ -100,14 +243,47 @@ party = {
...
+		economic_policy = planned_economy1
+		ideology = communist_social
+		religious_policy = pro_atheism
+		trade_policy = protectionism
+		war_policy = pro_military
+		start_date = 1961.1.1
+		end_date = 2006.2.4
+}
+party = {
+		name = "Québec solidaire (LN)"
```
Comment (settintotrieste) [inline CWE/common/countries/Quebec.txt:279]: No special characters please

File: CWE/history/countries/QUE - Quebec.txt [hunk 5]
```diff
@@ -84,7 +84,7 @@ schools = army_tech_school
 	non_state_culture_literacy = 0.3
 	civilized = yes
 	prestige = 24
-	ruling_party = party_socialist_0_QUE
+	ruling_party = "Parti Québécois"
```
Comment (settintotrieste) [inline CWE/history/countries/QUE - Quebec.txt:87]: No special characters please

Comment (ccfox) [reply CWE/history/countries/QUE - Quebec.txt]: Okay.

**Pull request merged**